plugins/inputs/vsphere/vsphere.go

```diff
@@ -202,6 +181,7 @@ func init() {
 			Timeout:                     config.Duration(time.Second * 60),
 			HistoricalInterval:          config.Duration(time.Second * 300),
 			DisconnectedServersBehavior: "error",
+			HTTPProxy:                   proxy.HTTPProxy{UseSystemProxy: true},
```

Comment:
should this be false to prevent a change in behavior?

Reply:
No as the current behavior is to use the system proxy settings in the vsphere client.